docs/source/client/actions/aliases.md

```diff
@@ -2,124 +2,64 @@
 
 A mechanism to redirect old URLs to new ones.
 
-When an object is moved (renamed or cut/pasted into a different location), the redirection storage will remember the old path. It is smart enough to deal with transitive references (if we have a -> b and then add b -> c, it is replaced by a reference a -> c) and circular references (attempting to add a -> a does nothing).
+When an object is moved (renamed or cut-and-pasted into a different location), the redirection storage will remember the old path. 
+It handles transitive references intelligently (for example a -> b, b -> c becomes a -> c), ignoring circular ones (for example attempting a -> a has no effect).
 
-## Get Aliases List
+## Get aliases list
 
-### Query function
+```{js:function} getAliasesListQuery
 
 Use the `getAliasesListQuery` function to get the query for fetching the aliases list.
 
-### Hook
+:hook: `useGetAliasesList`
+```
 
-Use the `useGetAliasesList` hook to get the aliases list.
+## Get aliases
 
-## Get Aliases
-
-### Query function
+```{js:function} getAliasesQuery(path)
 
 Use the `getAliasesQuery` function to get the query for fetching the aliases for a page.
 
-### Hook
-
-Use the `useGetAliases` hook to get the aliases for a page.
-
-### Parameters
-
-- **path**: string
-
-  - **Required:** Yes
-
-## Add Aliases for Multiple Pages
-
-### Mutation function
-
-Use the `createAliasesMutation` function to get the mutation for adding aliases for multiple pages.
-
-### Hook
-
-Use the `useCreateAliases` hook to add aliases for multiple pages.
+:arg string path: Description of the `path` parameter.
+:hook: `useGetAliases`
+```
 
-### Parameters
+## Add aliases for many pages
 
-- **data**: object
+```{js:function} createAliasesMutation(data)
 
-  - **Required:** Yes
-  - It can have the following fields:
+Use the `createAliasesMutation` function to get the mutation for adding aliases for many pages.
 
-    `items: object[]`:
+:arg array data: It can have the following fields:
+  - items: An array of objects with the following fields:
+    - path: string
+    - redirect_to: string
+    - [datetime]: string
+:hook: `useCreateAliases`
+```
```

Comment:
I don't think we have an easy and useful way to show data structures in narrative text through docstrings. Here's my latest attempt, but it is still unsatisfactory:

```markdown
## Add aliases for many pages

```{js:function} createAliasesMutation(data)

Use the `createAliasesMutation` function to get the mutation for adding aliases for many pages.

:arg array data: It can have the following fields:

                 `items`
                 : An array of objects with the following fields:

                     -   `path`: string
                     -   `redirect_to`: string
                     -   \[`datetime`\]: string

:hook: `useCreateAliases`
```

<img width="851" alt="Screenshot 2024-01-07 at 5 21 35 AM" src="https://github.com/plone/volto/assets/102112/9e992141-f80e-485a-9967-d1d8eed2f1a9">

That Markdown is horrible and difficult to write and maintain, and the result still does not look correct.

Perhaps we should instead adopt the practice of what we do in the Plone REST API, where we show the data structure to send as a `POST` request with an expected response?

https://6.docs.plone.org/plone.restapi/docs/source/endpoints/aliases.html#adding-url-aliases-in-bulk

@Hrittik20 what do you think?

Before going further, I think I want to chat with @sneridagh to discuss how we can both improve this package's documentation and better integrate with the Plone REST API docs. That conversation might yield a better direction. I ask for your patience until we can work through this. It might take until our next Volto Team meeting, which you are welcome to join. See https://discord.com/channels/786421998426521600/787308038050545666/1191701430642430054 for details.

I apologize that I did not realize that this would become more complicated, but I want to thank you for helping me come to this realization.

Reply:
@stevepiercy Absolutely, I agree with your suggestion to improve the documentation following the Plone REST API approach. Thanks for your assistance throughout this matter. Also, can you share the discord link again (link expired probably)?

Reply:
@Hrittik20 https://discord.gg/zFY3EBbjaj from https://plone.org/community/chat